aerogear/mobile-security-service
# feat(ui): Save changes to an app from the UI (AEROGEAR-8908)

## Motivation

https://issues.jboss.org/browse/AEROGEAR-8908

## What

Save the changed app versions to the database from the UI.

## Why

This is a requirement to meet acceptance criteria.

## How

Added a new function to data service and send the information to the API.

## Verification Steps
 
1. Go to the app view , e.g. `Home > App Name`.
2. Update the available fields on the table and click `Save`.
3. When the save is complete, the Save button should  become disabled again.
4. Refresh the page. The new changes should have persisted.

## Checklist:


## Progress


## Additional Notes
